src/Command/SyncRecipesCommand.php

```diff
@@ -80,6 +88,45 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $operations[] = new InstallOperation($pkg);
         }
 
+        if ($createEnvLocal = $force && file_exists($this->rootDir.'/.env') && file_exists($this->rootDir.'/.env.dist') && !file_exists($this->rootDir.'/.env.local')) {
+            $io->writeError('Renaming .env -> .env.local and .env.dist -> .env');
+            rename($this->rootDir.'/.env', $this->rootDir.'/.env.local');
+            $pipes = [];
+            proc_close(proc_open(sprintf('git mv .env.dist .env > %s 2>&1 || %s .env.dist .env', $win ? 'NUL' : '/dev/null', $win ? 'rename' : 'mv'), $pipes, $pipes, $this->rootDir));
+            if (file_exists($this->rootDir.'/phpunit.xml.dist')) {
+                touch($this->rootDir.'/.env.test');
+            }
+        }
+
         $this->flex->update(new UpdateEvent($force), $operations);
+
+        if ($force) {
+            $output = [
+                '',
+                '<bg=blue;fg=white>                                                    </>',
+                '<bg=blue;fg=white> Config files are now reset to their initial state. </>',
+                '<bg=blue;fg=white>                                                    </>',
+                '',
+                '  * Use <comment>git diff</comment> to inspect the changes.',
+                '',
+                '    Not all of them are relevant to your app: you now need to',
+                '    selectively add or revert them using e.g. a combination',
+                '    of <comment>git add -p</> and <comment>git checkout -p</>',
+                '',
+                '  * Use <comment>git checkout .</> to revert the changes.',
+                '',
+            ];
+
+            if ($createEnvLocal) {
+                $output[] = '    To revert the changes made to .env files, run';
```

Comment:
This is displayed only once when this is first applied. I think its important to provide the reverse path so that people can cancel and complete later.